ocaml/typing/printtyp.ml

```diff
@@ -1860,8 +1860,8 @@ let tree_of_type_decl id decl =
   in
   (* The algorithm for setting [lay] here is described as Case (C1) in
      Note [When to print jkind annotations] *)
```

Comment:
Ideally, that Note would be updated to match the new algorithm. But perhaps all of this is short-lived and so it doesn't matter much.

Reply:
Done